docs/core.yml

```diff
@@ -1274,6 +1274,104 @@ paths:
                         value:
                           type: string
 
+# ORGS - API CLIENTS
+
+  /api/v3/orgs:
+    post:
+      tags:
+      - orgs
+      summary: Create an organization
+      operationId: Org_Create
+      responses:
+        200:
+          description: A successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrgResponse_core'
+  /api/v3/orgs/{orgId}/block:
+    post:
+      tags:
+        - orgs
+      summary: Block an organization
+      operationId: OrgBlock_Post
+      parameters:
+        - name: orgId
+          in: path
+          description: The unique Code42 orgGuid of the org to return.
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: A successful response
+          content:
+            application/json:
+              schema:
+                type: object
+  /api/v3/orgs/{orgId}/unblock:
+    post:
+      tags:
+        - orgs
+      summary: Unblock an organization
+      operationId: OrgUnblock_Post
+      parameters:
+        - name: orgId
+          in: path
+          description: The unique Code42 orgGuid of the org to return.
```

Comment:
I know description really isn't used anywhere in the mock servers, but these should probably each match the action: `The unique Code42 orgGuid of the org to unblock.`